### ISSUE TITLE
E126 false positive

### ISSUE THREAD

**Issue by jianli** [PyCQA/pycodestyle#144]
Unresolved issue from #138. I'm creating a new issue since I'm unable to reopen that issue.

> E126 continuation line over-indented for hanging indent

``` python
{
    1:
        999999 if True
        else 0,
}
```

@florentx wrote that this would be better written with parentheses.

I responded:

> what's the rationale behind rewriting [it] with parentheses? The parentheses aren't syntactically necessary, and they don't make the expression more readable either.
